src/bcrypt/__init__.py

```diff
@@ -39,18 +39,19 @@ def gensalt(rounds=12, prefix=b"2b"):
     if prefix not in (b"2a", b"2b"):
         raise ValueError("Supported prefixes are b'2a' or b'2b'")
 
+    # TODO: is this the right min/max?
+    if rounds < 4 or rounds > 31:
+        raise ValueError("Invalid rounds")
+
     salt = os.urandom(16)
     output = _bcrypt.ffi.new("unsigned char[]", 30)
+    _bcrypt.lib.encode_base64(output, salt, len(salt))
```

Comment:
Right, I see we need our own base64 encoding in the C code to return the hashed password, but I think that this particular call can just use the standard library version. I guess there's an argument to be made for using this here so that we use the same encoding function everywhere, but it feels like to me it'd be better to just not expose the ``encode_base64`` function via cffi and just treat it as an internal implementation detail of the library, and to use the standard library code in Python.